object_test.go

```diff
@@ -1433,6 +1444,13 @@ func TestObject_Filter(t *testing.T) {
 		object.chain.assertNotFailed(t)
 	})
 
+	t.Run("nil func", func(t *testing.T) {
```

Comment:
Let's rename this to "invalid argument" and move to the end of the test.

Reply:
Changed format to the consistency of other tests